vtiger_connector_partner/models/res_company.py

```diff
@@ -95,7 +94,7 @@ def sync_vtiger_partner_vendor(self):
                         'name': res.get('vendorname'),
                         'email': res.get('email'),
                         'website': res.get('website'),
```

Comment:
We can create a new method and that will work on the query data to avoid the code duplication